src/System.Runtime.WindowsRuntime/src/System/IO/WindowsRuntimeStreamExtensions.cs

```diff
@@ -167,7 +167,7 @@ private static Stream AsStreamInternal(Object windowsRuntimeStream, Int32 buffer
             // There is already an adapter:
             if (adapterExists)
             {
-                Debug.Assert((adapter is BufferedStreamWrapper && ((BufferedStreamWrapper)adapter).UnderlyingStream is WinRtToNetFxStreamAdapter)
+                Debug.Assert((adapter is BufferedStream && ((BufferedStream)adapter).UnderlyingStream is WinRtToNetFxStreamAdapter)
```

Comment:
The .csproj file is referencing the System.IO implementation - the implementation always have this member.

Yes, the CoreCLR build is still using CoreLib version of BufferedStream. It is not easy to fix in corefx because of name collisions. I plan to remove BufferedStream from CoreLib and then fix it.